src/controller/ordersControllers.js

```diff
@@ -3,35 +3,35 @@ import orders from '../models/orders';
 
 
 export function postOrder (req, res) {
```

Comment:
#### Critical
You were advised to use `ES6`. Please remove every instances of `function` keyword and use the arrow function. i.e. This should be refactored to:
```javascript
export const postOrder = (req, res) => {
  // ....
}
```

Exceptions might be configuration files.

Reply:
Okay I will remove all instances of function keyword,
 thank you for suggesting a way

Reply:
I have implemented the feedback on this line, thank you